pkg/cvo/sync_worker.go

```diff
@@ -459,9 +459,14 @@ func (w *SyncWorker) Update(ctx context.Context, generation int64, desired confi
 	implicit, err := w.loadUpdatedPayload(ctx, work, cvoOptrName)
 	w.lock.Lock()
 	if err != nil {
-		// save latest capability settings if not first time through
+		// save override and capability changes if not first time through
 		if w.work != nil {
-			w.work.Capabilities = work.Capabilities
+			if !overridesEqual {
+				w.work.Overrides = config.Spec.Overrides
+			}
+			if !capabilitiesEqual {
```

Comment:
Do we need these qualifiers?  If they are equal, then `w.work.Capabilities = work.Capabilities` is a no-op, so you get the same behavior by performing the assignment unconditionally, right?  Same for overrides above.